packages/@stimulus/polyfills/index.js

```diff
@@ -15,3 +15,18 @@ if (typeof SVGElement.prototype.contains != "function") {
     return this === node || this.compareDocumentPosition(node) & Node.DOCUMENT_POSITION_CONTAINED_BY
   }
 }
+
+// From https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected#polyfill
+if (!("isConnected" in Node.prototype)) {
```

Comment:
@javan this was lifted directly from the [Node.isConnected MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected#polyfill), and is different from your proposed polyfill:

```js
if (!("isConnected" in Node.prototype)) {
  Object.defineProperty(Node.prototype, "isConnected", {
    get() {
      return document.documentElement.contains(this)
    }
  })
}
```

Do we have a preference for which we'd like to ship?

Reply:
Mine was quick sketch. This one looks solid.